diff --git a/node_modules/@stryker-mutator/vitest-runner/dist/src/vitest-test-runner.js b/node_modules/@stryker-mutator/vitest-runner/dist/src/vitest-test-runner.js
--- a/node_modules/@stryker-mutator/vitest-runner/dist/src/vitest-test-runner.js
+++ b/node_modules/@stryker-mutator/vitest-runner/dist/src/vitest-test-runner.js
@@ -126,10 +126,7 @@
         this.ctx.provide('hitLimit', options.hitLimit);
         this.ctx.provide('mutantActivation', options.mutantActivation);
         this.ctx.provide('activeMutant', options.activeMutant.id);
-        const dryRunResult = await this.run({
-            testIds: options.testFilter,
-            relatedFiles: [options.sandboxFileName],
-        });
+        const dryRunResult = await this.run({});
         const hitCount = this.readHitCount();
         const timeOut = determineHitLimitReached(hitCount, options.hitLimit);
         return toMutantRunResult(timeOut ?? dryRunResult);
diff --git a/node_modules/@stryker-mutator/vitest-runner/src/vitest-test-runner.ts b/node_modules/@stryker-mutator/vitest-runner/src/vitest-test-runner.ts
--- a/node_modules/@stryker-mutator/vitest-runner/src/vitest-test-runner.ts
+++ b/node_modules/@stryker-mutator/vitest-runner/src/vitest-test-runner.ts
@@ -207,10 +207,7 @@
     this.ctx!.provide('hitLimit', options.hitLimit);
     this.ctx!.provide('mutantActivation', options.mutantActivation);
     this.ctx!.provide('activeMutant', options.activeMutant.id);
-    const dryRunResult = await this.run({
-      testIds: options.testFilter,
-      relatedFiles: [options.sandboxFileName],
-    });
+    const dryRunResult = await this.run({});
     const hitCount = this.readHitCount();
     const timeOut = determineHitLimitReached(hitCount, options.hitLimit);
     return toMutantRunResult(timeOut ?? dryRunResult);
